src/analysis/retail/shaman/enhancement/modules/talents/HotHand.tsx

```diff
@@ -66,38 +70,64 @@ class HotHand extends Analyzer {
     );
   }
 
-  applyHotHand() {
+  applyHotHand(event: ApplyBuffEvent) {
     // on application both resets the CD and applies a mod rate
     this.spellUsable.endCooldown(TALENTS_SHAMAN.LAVA_LASH_TALENT.id);
     this.spellUsable.applyCooldownRateChange(TALENTS_SHAMAN.LAVA_LASH_TALENT.id, HOT_HAND.MOD_RATE);
+
+    this.hotHandActive.startInterval(event.timestamp);
+  }
+
+  refreshHotHand() {
+    // on refresh, we only need to reset the CD, since we've already applied the mod rate
+    this.spellUsable.endCooldown(TALENTS_SHAMAN.LAVA_LASH_TALENT.id);
   }
 
-  removeHotHand() {
+  removeHotHand(event: RemoveBuffEvent) {
     this.spellUsable.removeCooldownRateChange(
       TALENTS_SHAMAN.LAVA_LASH_TALENT.id,
       HOT_HAND.MOD_RATE,
     );
+
+    this.hotHandActive.endInterval(event.timestamp);
   }
 
   onLavaLashDamage(event: DamageEvent) {
     if (!this.selectedCombatant.hasBuff(SPELLS.HOT_HAND_BUFF.id)) {
       return;
     }
 
+    this.buffedCasts += 1
     this.buffedLavaLashDamage += calculateEffectiveDamage(event, HOT_HAND.INCREASE);
   }
 
+  get timePercentageHotHandsActive() {
+    return this.hotHandActive.totalDuration / this.owner.fightDuration;
+  }
+
+  get castsPerSecond() {
+    return this.buffedCasts / (this.hotHandActive.intervalsCount);
+  }
+
   statistic() {
     return (
       <Statistic
         position={STATISTIC_ORDER.OPTIONAL()}
         size="flexible"
+        tooltip={
+          <ul>  
+            <li>Gained buff {this.hotHandActive.intervalsCount} times ({formatPercentage(this.timePercentageHotHandsActive)}% uptime)</li>
+            <li>{this.buffedCasts} total Lava Lash casts with Hot Hand buff</li>
```

Comment:
Use `<SpellLink id={TALENTS_SHAMAN.LAVA_LASH_TALENT.id}/>` instead of writing talent name out

Reply:
I recommend using `<SpellLink id={TALENTS_SHAMAN.LAVA_LASH_TALENT}/>` instead of the `.id` version so you can save a request to the Blizz APIs. It renders faster and is generally correct.